packages/packages/locations/src/components/error-boundary.tsx

```diff
@@ -0,0 +1,29 @@
+import { Component, ReactNode } from 'react';
+
+interface Props {
+    children?: ReactNode;
+	fallback: ReactNode;
```

Comment:
fix lint

Reply:
Your eslint rule will fix it, for now Done